sys/include/net/gnrc/sixlowpan/internal.h

```diff
@@ -44,6 +47,24 @@ void gnrc_sixlowpan_dispatch_recv(gnrc_pktsnip_t *pkt, void *context,
 void gnrc_sixlowpan_dispatch_send(gnrc_pktsnip_t *pkt, void *context,
                                   unsigned page);
 
+
+/**
+ * @brief   Checks if packet fits over interface (and fragments if @ref
+ *          net_gnrc_sixlowpan_frag is available and required)
+ *
+ * @param[in] pkt                   The packet to fit. Must not be NULL.
+ * @param[in] orig_datagram_size    The original (uncompressed) datagram size.
+ *                                  Must be greater or equal to the length of
+ *                                  @p pkt as of `pkt->next` (i.e. without
+ *                                  the @ref gnrc_netif_hdr_t).
+ * @param[in] netif                 The interface to fit @p pkt over. Must not
+ *                                  be NULL.
+ * @param[in] page                  Current 6Lo dispatch parsing page
+ */
+void gnrc_sixlowpan_multiplex_by_size(gnrc_pktsnip_t *pkt,
+                                      size_t orig_datagram_size,
+                                      gnrc_netif_t *netif,
```

Comment:
```c
const gnrc_netif_t *netif,
```

Reply:
Mh... while in the current version of the code `netif` isn't changed, I can see a future version where `netif` is provided as to a `_send()` function which then again would require casting.